Sources/TuistCore/Graph/Graph.swift

```diff
@@ -497,6 +521,12 @@ public class Graph: Encodable {
 
     // MARK: - Fileprivate
 
+    fileprivate func isDynamicAndLinkable(node: PrecompiledNode) -> Bool {
```

Comment:
What about moving this method to `PrecompiledNode`?

Reply:
Sure, that makes it more reusable in other contexts 👍